packages/typespec-ts/test/commands/cadl-ranch-list.js

```diff
@@ -251,10 +251,10 @@ export const rlcTsps = [
     outputPath: "versioning/madeOptional",
     inputPath: "versioning/madeOptional"
   },
-  {
-    outputPath: "versioning/removed",
-    inputPath: "versioning/removed"
-  },
+  // {
+  //   outputPath: "versioning/removed",
+  //   inputPath: "versioning/removed"
+  // },
```

Comment:
disable  RLC versioning/removed as generating code contains interfaces with duplicate names. will create an issue for it.